### PR TITLE
Make Readme mirror CurseForge Page

### PR DESCRIPTION
**Pull request by RGFTheCoder**
Swaps the Description and Notes as it seems more fit for a readme

### CODE REVIEW

**Comment by RGFTheCoder** [comment]
Should the Donate and Social Media sections be padded at the start? If so, should they have equal padding as I currently set it, or should they have varied padding (so the donation options look centered)

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
...
+
+Chipped is a 1.16 building mod that expands on the variety of blocks you can use in-game. The mod offers overs 2000 unique building blocks, from stained glass variations all the way to concrete, prismarine and even gilded blackstone! Not only are all the textures hand-painted in the new vanilla style, but they also include some connected textures! Inspired by the popular Chisel mod, Chipped allows builders to have their full range of creativity, because every block deserves a friend!
+
+Chipped will aim to support all Minecraft versions 1.16+. We have only recently released, so making variations for all vanilla textures is our current priority. In the future, Chipped is planning to have compatibility with other popular mods.
+
+You are free to use this mod in your modpacks! Please just provide credit (:
+
+## Important Notes
+
+With the release of our newest update 1.05. We added a bunch of connected textures! To have these ctm textures work in-game, please download the mod found on the [Connected Textures Mod Curseforge page](https://www.curseforge.com/minecraft/mc-mods/chipped). We plan to release more cool blocks in future updates so stay tuned!
```
Comment (UpcraftLP) [inline README.md:15]: shouldn't this link to CTM instead?

**Review by UpcraftLP — CHANGES_REQUESTED** [review]
You missed one tiny detail (CTM link points to chipped), plus it'd probably be good to also add a separate link to the chipped curseforge page.

File: README.md (hunk 1 above)
Comment (terrariumearth) [reply README.md:15]: ill fix the link, it was an error on curse we realized just a bit ago, thank you guys!

**Comment by terrariumearth** [comment]
tysm!! <3

**Pull request merged**